site/en/docs/extensions/whatsnew/index.md

```diff
@@ -211,19 +242,33 @@ worker's script to import other modules.
 
 ### Chrome 91: `chrome.action.getUserSettings()` {: #chromeactiongetusersettings-available }
 
+April 2, 2021
+
 The new
 [`chrome.action.getUserSettings()`](/docs/extensions/reference/action/#method-getUserSettings)
 method allows extensions to determine if the user has pinned the extension to the main toolbar.
 
 ### Chrome 90: `chrome.scripting.removeCSS()` {: #chromescriptingremovecss-available }
 
+February 10, 2021
+
 The new [`chrome.scripting.removeCSS()`](/docs/extensions/reference/scripting/#method-removeCSS)
 method allows extensions to remove CSS that was previously inserted via
 [`chrome.scripting.insertCSS()`](/docs/extensions/reference/scripting/#method-insertCSS). It
 replaces [`chrome.tabs.removeCSS()`](/docs/extensions/reference/tabs/#method-removeCSS).
 
+### Chrome 90: Returning promises from `scripting.executeScript()` {: #m96-execute-script }
+
+February 24, 2021
+
+[`chrome.scripting.executeScript()`](/docs/extensions/reference/scripting/#method-executeScript)
+now supports returning promises. When a script evaluates to a promise, Chrome will wait for the
```

Comment:
```suggestion
now supports returning promises. When a method call evaluates with a promise, Chrome will wait for the
```

Reply:
This suggestion doesn't capture what we're trying to communicate. For context, `scripting.executeScript()` is similar to the [snippets](https://developer.chrome.com/docs/devtools/javascript/snippets/) feature in Chrome DevTools in that a script is executed and the final statement evaluated is the "result" of the execution. In DevTools, this result is logged to the console. In an `executeScript` call, it's stringified and passed back to the `executeScript`'s callback function as the [results](https://developer.chrome.com/docs/extensions/reference/scripting/#method-executeScript-callback) property. 

The suggested change implies any method invocation in the injected script that returns a promise will cause Chrome to await the resolution of that promise. Instead, what we're saying is that if the script evaluates to a promise (either by having the final statement evaluate to a promise or using the `func` property to inject a function that returns a promise), we will await the resolution of that promise, stringify the settled value, and pass it back to the `executeScript` callback. We illustrate this in the [Promises](https://developer.chrome.com/docs/extensions/reference/scripting/#promises) section of the scripting API documentation using an async function.

If you're still not happy with the current wording, perhaps we could change the final sentence, I'd suggest that we use the exact sentence from the scripting API doc: "If the resulting value of the script execution is a promise, Chrome will wait for the promise to settle and return its resulting value."

Reply:
That's actually a better statement.

I was not able to infer that behavior from the existing language. I suggested the change because I thought you were just being overly broad with your terminology. I thought it was analogous to someone using the word 'API' when they're talking about a method.

Reply:
Ack, using the sentence form the Scripting API docs.